src/constants/terminal.ts

```diff
@@ -1,5 +1,5 @@
 export const SUPPORTED_SHELLS = {
```

Comment:
Let's make the SUPPORTED_SHELLS a record of `<string, string[]>`

Reply:
ig it's is in this form
![image](https://user-images.githubusercontent.com/65275144/187884339-6771b4f2-9133-46cc-a6ec-75082637a7d5.png)

Reply:
ig its already resolved just added some more shells which many people use